pkg/registry/pod/rest/log.go

```diff
@@ -32,8 +32,8 @@ import (
 
 // LogREST implements the log endpoint for a Pod
 type LogREST struct {
```

Comment:
I'd rather not - I want to keep every I can in alphabetic order, as otherwise it's really hard to find anything in structures, as they grow in time.